Player/src/supplementary/experiments/Run.java

```diff
@@ -35,8 +37,8 @@ private static void calculateConfidence(Map<String, Integer> results, int draws,
         try {
             Process p = Runtime.getRuntime().exec(String.format("python%s scripts/confidence_calculator.py %d %d %d", System.getProperty("os.name").toLowerCase().contains("windows") ? "" : "3", results.get(ALGO_NAME), draws, results.get("MCTS")));
             BufferedReader pout = new BufferedReader(new InputStreamReader(p.getInputStream()));
-            {String s = null;; while ((s = pout.readLine()) != null) {System.out.println(""+s.stripTrailing());}}
-            BufferedReader perr = new BufferedReader(new InputStreamReader(p.getErrorStream())); {String s = null; while ((s = perr.readLine()) != null) {System.out.println("!>"+s.stripTrailing());}}
+            {String s = null;; while ((s = pout.readLine()) != null) {System.out.println(StringRoutines.cleanWhitespace(s));}}
```

Comment:
IMPORTANT NOTE I changed this because I still use Java 8 on my desktop, which doesn't have stripTrailing(). My cleanWhitespace function also cleans whitespace at the beginning though, not only trailing. So you might have to change this back if it's important for you.

Reply:
Same for the other Run files